src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball() {
+
+    }
```

Comment:
There is an unnecessary empty line after the class declaration and before the constructor, which should be removed to follow best practices.

Reply:
Removed empty line